server/src/main/java/io/crate/Constants.java

```diff
@@ -21,11 +21,16 @@
 
 package io.crate;
 
+import io.crate.metadata.pgcatalog.OidHash;
+import io.crate.user.User;
+
 public class Constants {
 
     public static final String DB_NAME = "crate";
     public static final int DB_OID = 0;
 
+    public static final int CRATE_USER_OID = OidHash.userOid(User.CRATE_USER.name());
+
```

Comment:
Maybe keep this local to the test? Not sure if we want to promote this outside of tests to depend on the oid of the user.

Reply:
sure, makes sense